lusid-sdk-python/tests/test_instrument_master.py

```diff
@@ -0,0 +1,230 @@
+import unittest
```

Comment:
Could you resolve any PEP8 formatting warning (ask me if you need me to show you how)

Reply:
done

Reply:
done